content/cybersecurity/concepts/spear-phishing/spear-phishing.md

```diff
@@ -0,0 +1,32 @@
+# Spear phishing
+
+Spear phishing is a type of social engineering scam that attempts to trick the recipient into providing confidential information to the attacker. Spear phishing is a more targeted cyber attack than phishing.
+
+## Spear phishing In-Use:
+
+The targeted nature of spear phishing makes it difficult to detect. For example, attackers who claim to be the CEO could trick finance executives into sending money to their bank account. An attacker might create messages that seem like IT is asking for more information. Deploying multi-factor authentication, strong password management policies, and training and awareness campaigns are good measures to help mitigate the risk.
+
+## Signs of spear phishing
+
+Spear phishing can have significant financial and reputational consequences for an organization, however there are signs to watch out for to stay protected against these threats:
+
+- _A sense of urgency_: spear phishing subject lines usually provoke an urgent response in the hope that the victim will act impulsively and engage.
+- _Poor imaging_: most companies will make sure to provide high-quality logos and images in their emails. A spear phishing email may contain grainy or blurry graphics.
+- _Links, addresses and domains_: look for discrepancies in email addresses, links and domain names to identify a potential spear phishing attempt. If there is a link, hover over it to reveal the URL. 
+
+## How to prevent spear phishing?
+
+A multi-faceted approach is the best way to prevent spear phishing. Email protection solutions can detect suspicious emails. Security awareness training is equally as important. Employees are often the first line of defense so it is very important that they are trained on how to detect suspicious cyber activity. Employees need to know that they should not open spam or unsolicited email from unknown parties and that they should never click on links in the email. 
+
+## Spear phishing vs. Phishing vs. Whaling:
+
+Spear phishing often is used interchangeably with phishing and whaling, but it is important to note the differences. Phishing is a more generic attempt to dupe a victim. It emphasizes quantity. Phishing emails or texts are usually sent to large groups of individuals as opposed to a targeted individual in a spear phishing campaign. Whaling prioritizes high-level targets, specifically the various chief officers of an organization. 
```

Comment:
You mention the comparison to whaling, but never define the term in opposition to spear phishing. There should be a sentence covering it.